sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/client.py

```diff
@@ -169,6 +169,8 @@ def __init__(self, hostname, **kwargs):
         self._mgmt_link_lock = threading.Lock()
         self._retry_policy = kwargs.pop("retry_policy", RetryPolicy())
         self._keep_alive_interval = int(kwargs.get("keep_alive_interval", 0))
+        self._keep_alive_interval = int(self._keep_alive_interval) if self._keep_alive_interval is not None else 0
```

Comment:
wondering how this line is different from the line above?

Reply:
ah to avoid if the user explicitly passes in None?

Reply:
nit: just to combine the 2 lines/remove redundancy for if keep_alive_internal is not passed and 0 is the result of 171, would this work the same?
```suggestion
        self._keep_alive_interval = int(kwargs.get("keep_alive_interval") or 0)
```

Reply:
although I guess int(0) is redundant too, nvm

Reply:
maybe just removing 0 as the default in line 171? but def not important

Reply:
plus `int(None)` would raise an Error